Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Confirm.hs

```diff
@@ -124,14 +132,19 @@ handler transporter req quote = do
           case routeInfo of
             Just route -> setExp (searchRequestKey $ getId ride.id) route 14400
             Nothing -> logDebug "Unable to get the key"
+          let destination = last booking.toLocation
```

Comment:
Lets use lastMaybe.

Reply:
then destination will be of `Maybe Location` type, fromloc is of type `Location`. Then will have to write one more update query supporting Maybe Loc